tutorials/tts/MixerTTS_FastPitch_Training.ipynb

```diff
@@ -0,0 +1,704 @@
+    "For more details about model, please refer to the original [paper](https://arxiv.org/abs/2110.03584).\n",
+    "\n",
+    "Mixer-TTS like most NeMo models is defined as a LightningModule, allowing for easy training via PyTorch Lightning, and parameterized by a configuration, currently defined via a yaml file and loading using Hydra.\n",
+    "\n",
+    "Let's take a look using NeMo's pretrained model and how to use it to generate spectrograms."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "75949279",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the MixerTTSModel\n",
+    "from nemo.collections.tts.models import MixerTTSModel\n",
+    "from nemo.collections.tts.models.base import SpectrogramGenerator\n",
+    "\n",
+    "# Let's see what pretrained models are available\n",
+    "print(MixerTTSModel.list_available_models())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0deb7390",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# We can load the pre-trained model as follows\n",
+    "pretrained_model = \"tts_en_lj_mixertts\"\n",
+    "model = MixerTTSModel.from_pretrained(pretrained_model)\n",
+    "model.eval();"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "af8154a3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# MixerTTSModel is a SpectrogramGenerator\n",
+    "assert isinstance(model, SpectrogramGenerator)\n",
+    "\n",
+    "# SpectrogramGenerators in NeMo have two helper functions:\n",
+    "#   1. parse(text: str, **kwargs) which takes string and produces a token tensor\n",
+    "#   2. generate_spectrogram(tokens: 'torch.tensor', **kwargs) which takes the token tensor and generates a spectrogram\n",
+    "# Let's try it out\n",
+    "tokens = model.parse(text=\"Hey, this produces speech!\")\n",
+    "spectrogram = model.generate_spectrogram(tokens=tokens)\n",
+    "\n",
+    "# Now we can visualize the generated spectrogram\n",
+    "# If we want to generate speech, we have to use a vocoder in conjunction to a spectrogram generator.\n",
+    "# Refer to the Inference_ModelSelect notebook on how to convert spectrograms to speech.\n",
+    "from matplotlib.pyplot import imshow\n",
+    "from matplotlib import pyplot as plt\n",
+    "%matplotlib inline\n",
+    "imshow(spectrogram.cpu().detach().numpy()[0,...], origin=\"lower\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82d93859",
+   "metadata": {},
+   "source": [
+    "### FastPitch\n",
+    "\n",
+    "FastPitch is  a fully-parallel text-to-speech model based on FastSpeech, conditioned on fundamental frequency contours. The model predicts pitch contours during inference. Conditioning on frequency contours improves the overall quality of synthesized speech. For more details about model, please refer to the original [paper](https://arxiv.org/abs/2006.06873).\n",
```

Comment:
This text is old, because NeMo's FastPitch implementation has Aligner and you didn't mention it, please, add this information here. And also add link to original implementation (https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/SpeechSynthesis/FastPitch), because Aligner integration was created by not NeMo team and information about Aligner is not covered in FastPitch paper.

Reply:
Move information about FastPitch after this text about Mixer-TTS:
```
Mixer-TTS is non-autoregressive model for mel-spectrogram generation. The model is based on the MLP-Mixer architecture adapted for speech synthesis. It contains pitch and duration predictors, with the latter being trained with an unsupervised TTS alignment framework. For more details about model, please refer to the original [paper](https://arxiv.org/abs/2110.03584).

Mixer-TTS like most NeMo models is defined as a LightningModule, allowing for easy training via PyTorch Lightning, and parameterized by a configuration, currently defined via a yaml file and loading using Hydra.
```